metricbeat/module/stan/fields.go

```diff
@@ -0,0 +1,36 @@
+// Licensed to Elasticsearch B.V. under one or more contributor
```

Comment:
I don't think this file should be here? Is there any reason for adding this? I think we should clean this up.